stock_reserve/model/stock_reserve.py

```diff
@@ -132,8 +132,9 @@ def release(self):
         """
         Release moves from reservation
         """
-        move_recs = self.move_id
-        move_recs.action_cancel()
+        for rec in self:
```

Comment:
Awesome, thanks! I learned about `mapped()` only after I had originally made this change.